man/deepsea-stage-0.7

```diff
@@ -44,7 +44,7 @@ The master minion will be patched.
 .RE
 .B Reboot
 .RS
-If DeepSea detects the need for a reboot, the master will reboot. 
+By default the master will not reboot. 
 This can be changed by adjusting
 
 .B stage_prep_master. See also deepsea-customization (7) 
```

Comment:
Here you could explicitly say something about setting "default-update-reboot" to trigger reboots automatically if necessary.